src/gui/editors/PianoRoll.cpp

```diff
@@ -3759,10 +3795,11 @@ void PianoRoll::recordAccompany()
 
 	m_pattern->addJournalCheckPoint();
 	m_recording = true;
-
 	if( m_pattern->getTrack()->trackContainer() == Engine::getSong() )
 	{
-		Engine::getSong()->playSong();
+		
+        Engine::getSong()->playPattern( m_pattern );
+        Engine::getSong()->playSong();
```

Comment:
I'm curious what this change does. My instinct says it would attempt to play both the pattern and the song when recording, which might cause the pattern to play twice in some cases? I don't know this code well.